### ISSUE TITLE
Single column for Likes by your followings

### ISSUE THREAD

**Issue by RB77** [TwidereProject/Twidere-Android#265]
I see you can add columns with likes by a single user but I wonder if it's possible to have a single column with the likes from all the users I want or at least with likes by all my followings.

Something like the "Activity" column in Tweetdeck.

**Comment by mariotaku** [comment]
It'd be possible if you are using [official keys](https://gist.github.com/mariotaku/5465786) since [0.3.0-dev151214](https://github.com/TwidereProject/Twidere-Android/releases/tag/0.3.0-dev151214). System-wide notifications will follow.

**Comment by RB77** [comment]
Thanks a lot!

I can't make it work, though.

I've Installed the app from your link and I used the api you provided and since there's one that says "Tweetdeck" I've tried that one as well.

I've tried both (twitter for android and TD) several times and after the change I delete the account and login again.

I also tried to delete all data and starting from scratch putting the apis in the config wizard but nothing.

The column is there but it starts loading and it stops after a while but nothing happens.

Every other column works normally.
